### DISCUSSION TITLE
What's the theme used in the tailwindcss-intellisense README's screenshots?

### DISCUSSION THREAD

**Discussion by alessandro-newzoo** [tailwindlabs/tailwindcss#12353] [General]
Hello,
I saw it was asked a couple of years ago and was wondering if it's still Tokyo Night?

I've installed it but while some colors are similar, it's definitely not the same :(

This is the SS I'm talking about (from https://github.com/tailwindlabs/tailwindcss-intellisense):

![image](https://github.com/tailwindlabs/tailwindcss/assets/47320294/9e774db4-7cbf-4170-a39f-aace667a3ee9)

and here's what `Tokyo Night` and `Tokyo Night Storm` look like my VS Code:
<img width="1129" alt="image" src="https://github.com/tailwindlabs/tailwindcss/assets/47320294/372c2245-a751-4ea0-920c-f78ecd9b4629">

<img width="1135" alt="image" src="https://github.com/tailwindlabs/tailwindcss/assets/47320294/cdf545ea-b9c5-457c-bb32-2c2846a3f0dd">

Thank you 🙏

**Comment by adhanji8** [comment]
I'm also curious about this. Anyone have any idea?
